### ISSUE TITLE
Error style when build with webpack

### ISSUE THREAD

**Issue by tannguyenit** [Expertizo/react-fb-image-grid#10]
When i use loader of webpack i get problem 
![Error when build with webpack](https://user-images.githubusercontent.com/24540921/48115531-bd602780-e295-11e8-9b39-a3685a1670c9.png)

The reason it does is because all URLs are rebased first and if you don't pass import as a url(...) then it does not get rebased, thus finds the file in the current folder.
You should replace to 

@import url('https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css');

@import url('https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css');

Please resolve it for me. Thanks you

**Comment by kashifsulaiman** [comment]
@tannguyenit could you verify about this solution? I mean't have you tried doing like this and the webpack worked successfully?

**Comment by tannguyenit** [comment]
Yep, I change @import "link" to @import url("link") then webpack worked successfully

**Comment by keyhole425** [comment]
@tannguyenit Hey there, any chance you could share your webpack config?  I cannot get this to work no matter what is attempted. Thanks!

**Comment by tannguyenit** [comment]
> @tannguyenit Hey there, any chance you could share your webpack config? I cannot get this to work no matter what is attempted. Thanks!

https://gist.github.com/tannguyenit/a3663dce61d617c414477961ce3662e3#file-config-webpack
Here a my webpack config

**Comment by tannguyenit** [comment]
> @tannguyenit Hey there, any chance you could share your webpack config? I cannot get this to work no matter what is attempted. Thanks!

https://gist.github.com/tannguyenit/a3663dce61d617c414477961ce3662e3#file-config-webpack
Here my webpack config

**Comment by littlefroginnovations** [comment]
Next.JS

Failed to compile
./node_modules/react-image-lightbox/style.css
Global CSS cannot be imported from within node_modules.
Read more: https://err.sh/next.js/css-npm
Location: node_modules/react-fb-image-grid/es/components/Modal.js
